index.html

```diff
@@ -2238,19 +2242,18 @@ <h3>
           </p>
           <p>
             The <dfn>binaryType</dfn> attribute can take one of the values of
-            <dfn data-dfn-for="">BinaryType</dfn>. When a
+            <a data-link-for="">BinaryType</a>. When a
```

Comment:
Remove empty data-link-for attribute?

Reply:
That was needed to reset the `data-link-for="PresentationConnection"` attribute of the enclosing `<div>`, because Respec would have linked to `PresentationConnection.binaryType` otherwise (Respec is unfortunately not case sensitive there and doesn't distinguish between `binaryType` and `BinaryType`)

That said, it's probably more readable to drop the `data-link-for` attribute from the enclosing `<div>`, and rather flag the couple of references to `PresentationConnection.binaryType` in that section with a `data-link-for="PresentationConnection"` attribute. I just pushed an update for that.

Reply:
Thanks for the explanation!